tests/factories.py

```diff
@@ -91,10 +91,8 @@ def uri(stub):
         return "http://example.com/document_{n}".format(n=stub.random_number)
 
     @factory.LazyAttribute
-    def target(stub):
+    def target_selectors(stub):
```

Comment:
Ditto here. This is now returning a data structure that isn't compatible with the one returned by `target_selectors` on a real `Annotation` object.